lib/lualink.c

```diff
@@ -537,28 +543,71 @@ void Lua_crowbegin( void )
 {
     printf("init()\n"); // call in C to avoid user seeing in lua
     lua_getglobal(L,"init");
-    if( lua_pcall_protected(L,0,0,0) != LUA_OK ){
-        Caw_send_luachunk( (char*)lua_tostring( L, -1 ) );
-        lua_pop( L, 1 );
+    if( Lua_call_usercode(L,0,0) != LUA_OK ){
+        Caw_send_luachunk("error running init()");
```

Comment:
This line is a reversion. It changed to the `lua_tostring` call in a recent commit to get the actual error message from the lua env.

Reply:
The Lua error always gets reported with `Caw_send_luachunk` and `_printf` in `Lua_handle_error`, so sending it here would wind up printing it twice.

Reply:
Ahh cool. I'm still thinking in the previous mental model where the 'call' doesn't handle errors directly.